runtime/src/lib.rs

```diff
@@ -306,13 +308,15 @@ parameter_types! {
     pub const DemocracyPreimageByteDeposit: Balance = balance!(0.000002); // 2 * 10^-6, 5 MiB -> 10.48576 XOR
     pub const DemocracyMaxVotes: u32 = 100;
     pub const DemocracyMaxProposals: u32 = 100;
+    pub const DemocracyMaxDeposits: u32 = 100; // todo
+    pub const DemocracyMaxBlacklisted: u32 = 100; // todo
```

Comment:
Should we set some other values?
Lets decide and update values or remove comments

Reply:
It's the same values as in polkadot. I think it's ok to keep it